x-pack/packages/security-solution/features/src/cases/v2_features/kibana_sub_features.ts

```diff
@@ -0,0 +1,178 @@
+
+/**
+ * Sub-features that will always be available for Security Cases
+ * regardless of the product type.
+ */
+export const getCasesBaseKibanaSubFeatureIdsV2 = (): CasesSubFeatureId[] => [
+  CasesSubFeatureId.deleteCases,
+  CasesSubFeatureId.casesSettings,
+  CasesSubFeatureId.createComment,
+  CasesSubFeatureId.reopenCase,
+];
+
+/**
+ * Defines all the Security Solution Cases subFeatures available.
+ * The order of the subFeatures is the order they will be displayed
+ */
+export const getCasesSubFeaturesMapV2 = ({
+  uiCapabilities,
+  apiTags,
+  savedObjects,
+}: CasesFeatureParams) => {
+  const deleteCasesSubFeature: SubFeatureConfig = {
+    name: i18n.translate('securitySolutionPackages.features.featureRegistry.deleteSubFeatureName', {
+      defaultMessage: 'Delete',
+    }),
+    privilegeGroups: [
+      {
+        groupType: 'independent',
+        privileges: [
+          {
+            api: apiTags.delete,
+            id: 'cases_delete_v2',
+            name: i18n.translate(
+              'securitySolutionPackages.features.featureRegistry.deleteSubFeatureDetails',
+              {
+                defaultMessage: 'Delete cases and comments',
+              }
+            ),
+            includeIn: 'all',
+            savedObject: {
+              all: [...savedObjects.files],
+              read: [...savedObjects.files],
+            },
+            cases: {
+              delete: [APP_ID],
+            },
+            ui: uiCapabilities.delete,
+          },
+        ],
+      },
+    ],
+  };
+
+  const casesSettingsCasesSubFeature: SubFeatureConfig = {
+    name: i18n.translate(
+      'securitySolutionPackages.features.featureRegistry.casesSettingsSubFeatureName',
+      {
+        defaultMessage: 'Case settings',
+      }
+    ),
+    privilegeGroups: [
+      {
+        groupType: 'independent',
+        privileges: [
+          {
+            id: 'cases_settings_v2',
+            name: i18n.translate(
+              'securitySolutionPackages.features.featureRegistry.casesSettingsSubFeatureDetails',
+              {
+                defaultMessage: 'Edit case settings',
+              }
+            ),
+            includeIn: 'all',
+            savedObject: {
+              all: [...savedObjects.files],
+              read: [...savedObjects.files],
+            },
+            cases: {
+              settings: [APP_ID],
+            },
+            ui: uiCapabilities.settings,
+          },
+        ],
+      },
+    ],
+  };
+
+  /* The below sub features were newly added in v2 (8.16) */
+
+  const casesAddCommentsCasesSubFeature: SubFeatureConfig = {
+    name: i18n.translate(
+      'securitySolutionPackages.features.featureRegistry.addCommentsSubFeatureName',
+      {
+        defaultMessage: 'Add comments',
+      }
+    ),
+    privilegeGroups: [
+      {
+        groupType: 'independent',
+        privileges: [
+          {
+            api: apiTags.all,
+            id: 'create_comment',
```

Comment:
@azasypkin  - These are new permissions, so therefore no `_v2` label. Would you consider it good practice to `_v2` all the ids even if they're net new features? I think either way it _can_ be confusing, but would love to settle on some consistent pattern.

Reply:
Great, removing the `_v2` references for existing subprivileges